site/docs/numbers/errors.mdx

```diff
@@ -1195,7 +1195,7 @@ image: '@site/static/img/bw-icon.svg'
 | 17000 | Lsr submission order payload required. |
 | 400 | Lsr order doesn't pass validation. Please check your input parameters. |
 | 17002 | Order contains invalid tns: %s. |
-| 17003 | Order contains empty TN list (or it's not present). |
+| 17003 | Order contains empty tn list or its not present. |
```

Comment:
I changed it because in Iris in respective error code enum it was "...tn list or its not present."
<img width="797" alt="Screenshot 2022-09-28 at 15 14 05" src="https://user-images.githubusercontent.com/5918477/192775900-4773584b-7fe8-4551-8692-c8fc443fedfa.png">